add clarification about what versions of Yarn are compatible with the project (#136325)
elastic/kibana: dev_docs/getting_started/setting_up_a_development_env.mdx

npm install -g yarn
```

> If you already have Yarn installed, make sure its version is 1.22.19 or newer. Versions 2+ of Yarn are incompatible with this project.

Reviewer: ```suggestion
If you have Yarn installed, make sure you are using 1.22.19 or later. 2.0 and later are unsupported.
```

> In order to set your previously installed Yarn to a compatible version (for example 1.22.19) run

Reviewer: ```suggestion
. If you are using an unsupported Yarn version, run:
+
```

> ```sh

Reviewer: ```suggestion
[source,text]
--
```

Author: Hey, @KOTungseth . I don't see any ocurrence of [source,text] in *.mdx files on the Kibana repo. Only in *.asciidoc. Sorry for the rookie question, but what would it do in *.mdx files?

Reviewer: My apologies! You're right. Please ignore my comment.

yarn set version 1.22.19

> ```

Reviewer: ```suggestion
--
```

> Finally, bootstrap Kibana and install all of the remaining dependencies:

Reviewer: ```suggestion
. Bootstrap {kib}, then install the remaining dependencies:
```

Author: Same doubt here. `{kib}` placeholders are only found on *.asciidocs. Should I ignore and resolve this as well?

Reviewer: Yes, please ignore :)
